cannon/main.go

```diff
@@ -23,17 +22,8 @@ func main() {
 		cmd.WitnessCommand,
 		cmd.RunCommand,
 	}
-	ctx, cancel := context.WithCancel(context.Background())
-
-	c := make(chan os.Signal, 1)
-	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
-	go func() {
-		for {
-			<-c
-			cancel()
-			fmt.Println("\r\nExiting...")
-		}
-	}()
+	// This used to punt stdout on an interrupt, should that be a default behaviour?
```

Comment:
cc @Inphi

Reply:
It's OK to remove stdout write on interrupt. The interrupt error will surface during vm execution (cannon/cmd/run.go) and we'll see an error printed there.